Packs/SuspiciousDomainHunting/Layouts/layoutscontainer-Suspicious_Domain_Hunting_Incident_Layout.json

```diff
@@ -0,0 +1,1592 @@
+                                "id": "fc261d60-78c9-11ee-825c-270ac8ef585b",
+                                "index": 0,
+                                "name": "Resolve As False Positive",
+                                "scriptId": "Builtin|||closeInvestigation",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {},
+                                "buttonClass": "warning",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "150cc770-78ca-11ee-825c-270ac8ef585b",
+                                "index": 1,
+                                "name": "Run !Domain",
+                                "scriptId": "VirusTotal (API v3)|||domain",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {
+                                    "extended_data": {
+                                        "simple": "",
+                                        "userMarkedRequired": true
+                                    }
+                                },
+                                "buttonClass": "warning",
+                                "endCol": 2,
+                                "fieldId": "",
+                                "height": 44,
+                                "id": "231eaa40-7979-11ee-98e4-c5340a9b9a3f",
+                                "index": 2,
+                                "name": "Run !url",
+                                "scriptId": "Panorama|||url",
+                                "sectionItemType": "button",
+                                "startCol": 0
+                            },
+                            {
+                                "args": {
+                                    "body": {
+                                        "simple": "Dear ${Domain.Whois.Registrar.Name.[0]},\n\nI am writing to report abusive activity on the domain ${incident.domain}. \nThis activity is in violation of your terms of service or acceptable use policy.\nI kindly request your prompt investigation and necessary action to address this abuse, which may include suspending the domain.\n\nYour swift response and cooperation in resolving this matter are greatly appreciated.\n\nSincerely,\n${inputs.contactEmail}"
+                                    },
+                                    "subject": {
+                                        "simple": "Report of Abuse ${incident.domain}"
+                                    },
+                                    "to": {
+                                        "simple": "oitzhak@paloaltonetworks.com"
```

Comment:
remove the mail address   "simple": "oitzhak@paloaltonetworks.com"
need to change it to the Domain Registrar email address